miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/AttachmentsTag.java

```diff
@@ -0,0 +1,38 @@
+package uk.ac.bbsrc.tgac.miso.webapp.util.form;
+
+import java.util.stream.Collectors;
+
+import org.springframework.web.servlet.tags.RequestContextAwareTag;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Attachable;
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+
+public class AttachmentsTag extends RequestContextAwareTag {
+
+  private static final long serialVersionUID = 1L;
+
+  private Object item;
+
+  @Override
+  protected int doStartTagInternal() throws Exception {
+    Attachable item = (Attachable) this.item;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "item" which hides the field declared at line 16. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)